docs - instructions for running development branch (#19926)
metabase/metabase: docs/developers-guide/dev-branch-docker.md

# How to run a development branch of Metabase using Docker

If you want to run a branch of Metabase that's currently in development, the easiest way to get started is to use a pre-built Docker image. You can also [compile Metabase yourself](build.md).

If you're looking to download and run the latest official open source version of Metabase, check the [operations guide](/docs/operations-guide/installing-metabase.md). 

## Installing Docker

The only thing you’ll need to get started is Docker itself.

- [Install Docker Desktop](https://www.docker.com/products/docker-desktop)

OR

- If you like to install things via Homebrew:

```bash
brew install docker
```
Once Docker is installed, you’re ready to go.

> ## Run an in development branch to test or verify features

Reviewer: ```suggestion
## Run a development branch to test or verify features
```
